ddtrace/contrib/internal/botocore/patch.py

```diff
@@ -229,4 +232,29 @@ def patched_api_call_fallback(original_func, instance, args, kwargs, function_va
             raise
         else:
             core.dispatch("botocore.patched_api_call.success", [ctx, result])
+            _prepare_span_pointers_from_successful_response(
```

Comment:
Is there a good reason not to include this logic in the above `botocore.patched_api_call.success` event handler? I think that's what I'd do.

Reply:
ok, i've done that, but we end up with even more botocore-related code in `_trace/utils.py`. maybe this means we need a `_trace/botocore/utils`? but then why not use the `contrib/internal/botocore` space?

Reply:
after a discussion on slack, i think i understand what the process is here.

our trace handlers file is free to do what it needs with botocore parameters and responses. but a tracer engineer should not have to worry about *how* those parameters and responses are actually wired into the botocore apis.

i've created a new `ddtrace._trace.utils_botocore.*` package with an initial  `span_pointers` modules in it, but i plan to follow up this PR with a cleanup to move the rest of the botocore-specific code out of `ddtrace._trace.utils` into `ddtrace._trace.utils_botocore` for clarity.

Reply:
things are now rearranged.